app/experimenter/nimbus-ui/src/components/PreviewURL/index.tsx

```diff
@@ -19,24 +20,39 @@ const CopyableURL: React.FC<CopyableURLProps> = ({
   branch,
   collection,
 }) => {
+  // This about: url cannot be linked from web content, so help with auto-copy.
   let url = `about:studies?optin_slug=${experimentSlug}&optin_branch=${branch}`;
   const clipboard = navigator.clipboard;
   if (collection) {
     url = `${url}&optin_collection=${collection}`;
   }
```

Comment:
Not exactly part of your PR, but you could make `url` a ternary:

```
const url = collection ? `${url}&optin_collection=${collection}` : `about:studies?optin_slug=${experimentSlug}&optin_branch=${branch}`;
```

Reply:
`url` wouldn't be defined for the "true expression"